<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateTime</class>
 <widget class="QWidget" name="DateTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DateTime</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="WidgetMain" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#WidgetMain{
background:#FFFFFF;
}
QWidget#widgetDate,#widgetTime{
border-bottom:1px solid #A7B7CE;
}

QTimeEdit,QDateEdit{
border:1px solid #B9D2E2;
color:#43464B;
font-size:12pt;
}

QPushButton#btnYes{
border:0px;
color:#0092FF;
font-size:11pt;
background: transparent;
}
QPushButton#btnCancel{
border:0px;
color:#34445B;
font-size:11pt;
background: transparent;
}

/**上层年月导航栏**/
QCalendarWidget QWidget#qt_calendar_navigationbar{ 
background-color: #FFFFFF;
height:30px;
}

/**年月选择按钮**/
QToolButton#qt_calendar_monthbutton,#qt_calendar_yearbutton{
color:#43464B;
font-size:11pt;
border:0px;
background-color: #FFFFFF;
}
QToolButton#qt_calendar_monthbutton:hover,#qt_calendar_yearbutton:hover{
color:#43464B;
font-size:11pt;
height:15px;
background-color: #FFFFFF;
}
/**设置月份选择下拉按钮居中**/
QToolButton#qt_calendar_monthbutton::menu-indicator{
subcontrol-origin: padding;
subcontrol-position: center right;
right: 3px;
width: 10px;
}
/**月份前 后按钮**/
QToolButton#qt_calendar_prevmonth{
icon-size:20px 20px;
border:0px;
qproperty-icon:url(:/month-earlier.png);
background: transparent;
}
QToolButton#qt_calendar_nextmonth{
icon-size:20px 20px;
border:0px;
qproperty-icon:url(:/month-later.png);
background: transparent;
}

QCalendarWidget QMenu{
width:100px;
color:#43464B;
left:10px;
font-size:10pt;
background-color:#FFFFFF;
border:1px solid #EFF4F8;
}

QCalendarWidget QMenu QAction{
width: 100px;
color: #43464B;
font-size: 12pt;
background-color: #FFFFFF;
}
QCalendarWidget QMenu QAction:hover{
color:#FFFFFF;
background-color: #0092FF;
}
QCalendarWidget QWidget{ 
alternate-background-color: #FFFFFF;
border-bottom:1px solid #EFF4F8;
}
QHeaderView#qt_scrollarea_vcontainer{
background-color: #FF1111;
}
QCalendarWidget QAbstractItemView:enabled 
{
font-size:14px; 
color: #34445B; 
selection-background-color: #0092FF; 
selection-color: #FFFFFF;
}
/* days in other months */
QCalendarWidget QAbstractItemView:disabled { 
color:#C7C7C7; 
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QWidget" name="widgetDate" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QDateEdit" name="dateEdit">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>32</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>10000</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::NoButtons</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCalendarWidget" name="calendarWidget">
               <property name="minimumSize">
                <size>
                 <width>240</width>
                 <height>230</height>
                </size>
               </property>
               <property name="sizeIncrement">
                <size>
                 <width>240</width>
                 <height>230</height>
                </size>
               </property>
               <property name="selectionMode">
                <enum>QCalendarWidget::SingleSelection</enum>
               </property>
               <property name="verticalHeaderFormat">
                <enum>QCalendarWidget::NoVerticalHeader</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widgetTime" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>10</number>
             </property>
             <item>
              <widget class="QTimeEdit" name="timeEdit">
               <property name="minimumSize">
                <size>
                 <width>240</width>
                 <height>32</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::NoButtons</enum>
               </property>
               <property name="displayFormat">
                <string>H:mm:ss</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="spacing">
                <number>10</number>
               </property>
               <property name="topMargin">
                <number>2</number>
               </property>
               <property name="bottomMargin">
                <number>2</number>
               </property>
               <item>
                <widget class="QWidget" name="widgetHour" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>140</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="sizeIncrement">
                  <size>
                   <width>40</width>
                   <height>140</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widgetMin" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>140</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="sizeIncrement">
                  <size>
                   <width>40</width>
                   <height>140</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>calendarWidget</sender>
   <signal>clicked(QDate)</signal>
   <receiver>dateEdit</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calendarWidget</sender>
   <signal>activated(QDate)</signal>
   <receiver>dateEdit</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
